src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -227,32 +228,32 @@ public boolean isDependentArgumentAllowed(final Class<?> predecessorClass) {
     @Override
     public void validateArguments() {
```

Comment:
@magicDGS Can you remind me if there is a use-case you have in mind for that ? I seem to remember thinking you had cases where you wanted to *reduce* visibility of builtin arguments. Is there a case where you need to add additional arguments that need validation ? Just want to understand the context.